### PR TITLE
fixed bugs

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**
I am not sure how this worked before, but we were not creating the right processor for the private topics

### CODE REVIEW

File: sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs [hunk 1]
```diff
@@ -56,35 +56,37 @@ private ServiceBusSender GetServiceBusSender()
         return sender;
     }
 
-    internal ServiceBusProcessor GetServiceBusProcessor()
+    internal ServiceBusProcessor GetServiceBusProcessor(string id)
     {
         MessagingServices messagingServices = this;
-        ServiceBusProcessor sender = _cm.Subclients.Get(() => messagingServices.CreateProcessor());
+        ServiceBusProcessor sender = _cm.Subclients.Get(() => messagingServices.CreateProcessor(id), id);
         return sender;
```
Comment (jsquire) [inline sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs:63]:
```suggestion
        ServiceBusProcessor processor = _cm.Subclients.Get(() => messagingServices.CreateProcessor(id), id);
        return processor;
```

File: sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs [hunk 2]
```diff
@@ -56,35 +56,37 @@ private ServiceBusSender GetServiceBusSender()
...
-        ClientConnectionOptions connection = _cm.GetConnectionOptions(typeof(ServiceBusSender));
+        ClientConnectionOptions connection = _cm.GetConnectionOptions(typeof(ServiceBusProcessor), id);
+        string[] topicAndSubscription = connection.Id.Split('/');
+
         ServiceBusProcessor processor = client.CreateProcessor(
-            connection.Id,
-            "cm_servicebus_subscription_private",
+            topicAndSubscription[0],
+            topicAndSubscription[1],
             new() { ReceiveMode = ServiceBusReceiveMode.PeekLock, MaxConcurrentCalls = 5 });
```
Comment (jsquire) [inline sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs:90]:
```suggestion
            new() { MaxConcurrentCalls = 5 });
```

PeekLock is the default.  There's no reason to explicitly specify it.

**Review by christothes — APPROVED** [review]

File: sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs (hunk 1 above)
Comment (KrzysztofCwalina) [reply sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs]: fixed

File: sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs (hunk 2 above)
Comment (KrzysztofCwalina) [reply sdk/cloudmachine/Azure.CloudMachine/src/CoreServices/MessagingServices.cs]: fixed

**Pull request merged**